test/SlowTests/SlowTests.csproj

```diff
@@ -197,6 +197,9 @@
     <None Update="SchemaUpgrade\Issues\SystemVersion\after_from14.zip">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="SchemaUpgrade\Issues\SystemVersion\CompareExchange_Expiration_RavenData_v5.2_new_from50000.zip">
```

Comment:
There is no file like that added in this PR

Reply:
removed